pkgs/development/libraries/onnxruntime/default.nix

```diff
@@ -74,10 +88,27 @@ let
     rev = "refs/tags/v1.14.1";
     hash = "sha256-ZVSdk6LeAiZpQrrzLxphMbc1b3rNUMpcxcXPP8s/5tE=";
   };
+
+   cutlass = fetchFromGitHub {
+    owner = "NVIDIA";
+    repo = "cutlass";
+    rev = "v3.0.0";
+    sha256 = "sha256-YPD5Sy6SvByjIcGtgeGH80TEKg2BtqJWSg46RvnJChY=";
+   };
```

Comment:
Don't like this but out of scope, tracking in https://github.com/NixOS/nixpkgs/issues/289263